frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -1103,6 +1141,17 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
           </Row>
         ) : null}
 
+        {this.props.controlMode === ControlModeEnum.VIEW && this.props.isAdminOrManager ? (
```

Comment:
Is there a reason why this is only allowed from view mode? I cannot read from the issue description that this was a requirement and was wondering why you introduced this restriction.
I guess annotations would complicate this functionality, because volume annotation layers would need to be filtered out and/or replaced by their fallback layers :thinking: 
@normanrz Do you think it's sufficient to enable this for viewed datasets, only?

Reply:
> I guess annotations would complicate this functionality, because volume annotation layers would need to be filtered out and/or replaced by their fallback layers 

Excactly that was my thought :+1: (Additionally norman mentioned to me that view mode only should be fine)